### ISSUE TITLE
Compilation fail

### ISSUE THREAD

**Issue by tsulh** [CleverRaven/Cataclysm-DDA#19918]
Compiler: gcc (Debian 5.3.1-14) 5.3.1 20160409

Cataclysm fails to build from source.

```
src/item.cpp: In member function ‘std::__cxx11::string item::display_name(unsigned int) const’:
src/item.cpp:2355:14: error: ‘side’ is not a class, namespace, or enumeration
         case side::BOTH:
              ^
src/item.cpp:2357:14: error: ‘side’ is not a class, namespace, or enumeration
         case side::LEFT:
              ^
src/item.cpp:2360:14: error: ‘side’ is not a class, namespace, or enumeration
         case side::RIGHT:
              ^
src/item.cpp:2354:12: error: enumeration value ‘BOTH’ not handled in switch [-Werror=switch]
     switch (get_side()) {
            ^
src/item.cpp:2354:12: error: enumeration value ‘LEFT’ not handled in switch [-Werror=switch]
src/item.cpp:2354:12: error: enumeration value ‘RIGHT’ not handled in switch [-Werror=switch]
```

0.C-19534-gec2b8d9bbe (2 days ago) compiles for me.

**Comment by keyspace** [comment]
Probably compiler-related. Line 2351 has `std::string side;` declaration; `side` is also an enum ir `bodypart.h`.

**Comment by keyspace** [comment]
Origin: bd38afde275c3d7f44680742fc135ee9ab5fc788, multi-merged in 0ed06bd28455b239fbda1b09734775dfb80241e7 (today), was part of #19671.

**Comment by keyspace** [comment]
Note: `std::string side` also present in `player.h` and `player.cpp`.

**Comment by codemime** [comment]
Closed in #19916.
